docs/DXventures/Inquiries.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: default
+title: Inquiries
+parent: DXventures
+nav_order: 2
+---
+
+# Inquiries
+
+___
+
+
+Looking to make an inquiry in private?
+
+Contact Sky, DXventures lead, on [Daotalk](https://daotalk.org/u/sky/) or [Keybase](https://keybase.io/skymine).
```

Comment:
I think inquires should be handled differently. I don't think it should be in table of contents and displayed as subcategory. 
![Screenshot 2021-04-20 at 15 12 13](https://user-images.githubusercontent.com/33226956/115401538-cab1c800-a1ea-11eb-862c-1af553c52b1b.png)

Reply:
Agreed! 

The goal here was to have a base level that can be expanded upon alongside the review build. It doesn't quite make sense to have currently, and would be simple to create down the line regardless. Will remove inquiries subcategory, and consolidate to the DXventures category!